src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateServiceTest.kt

```diff
@@ -217,21 +222,21 @@ class ReleaseDateServiceTest {
     fun `should return cut-off date as 2018-03-14 when job execution date is 2018-03-12`() {
       val now = createClock("2018-03-12T00:00:00Z")
       val cutOffDate = service.getCutOffDateForLicenceTimeOut(now)
-      assertTrue(cutOffDate.isEqual(LocalDate.parse("2018-03-14")))
```

Comment:
Adjusted the tests as when the initial sequence was created using the old logic, it did not drop the first value which have been the date that was sent in.

Reply:
Chat with Andy confirms, like previously, the config working days was incorrect and should be 2 working days and not 3 so these tests should be kept the same.